src/main/java/de/tum/in/www1/artemis/web/rest/tutorialgroups/TutorialGroupDateUtil.java

```diff
@@ -55,6 +55,26 @@ public static boolean isIso8601TimeString(String timeString) {
         }
     }
 
+    /**
+     * Check if a string follows the ISO 8601 format for dateTime
+     *
+     * @param timeString the string to check
+     * @return true if the string follows the ISO 8601 format for time or if null
+     */
+    public static boolean isIso8601DateTimeString(String timeString) {
```

Comment:
Can we extract this in a general date util class (similar to our client)?
I think this might be a useful check in the future.

For reference take a look at `Objects.requireNonNull(Object o)`.

Reply:
Where would be the appropriate place for such a class? I can't seem to find any other "global" util classes on the server.

Reply:
We have the packages `de.tum.in.www1.artemis.web.rest.util` or `de.tum.in.www1.artemis.service.util` for example.